tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -261,4 +261,9 @@ Check Cards Details Are Correct
         ${badges_titles}=  Check Card Badges And Return Titles  card_locator=${card_xp}  app_id=${application_id}  expected_data=${expected_data}
         Check Card Image  card_locator=${card_xp}  app_id=${application_id}  expected_data=${expected_data}
         Check Get Started Sidebar  card_locator=${card_xp}  card_badges=${badges_titles}  app_id=${application_id}  expected_data=${expected_data}
-    END
\ No newline at end of file
+    END
+
+Get Link Elements
```

Comment:
Since this keyword returns only Documentation links (correct me if I am wrong), maybe it would be better to name it `Get Documentation Links Elements` or something like that

Reply:
yes, I will rename it